### ISSUE TITLE
TAStudio performance bug since 2.2.2

### ISSUE THREAD

**Issue by meshuggahtas** [TASVideos/BizHawk#1310]
After the release of BizHawk 2.2.2, I have a huge performance drop each time I'm using TAStudio.

This largely depends on what action you try on which controls. The most laggiest one is handling the piano roll obviously and nearly anything that relates to it (number of columns, window size, frame colorness, etc). The most lightest one is simply pressing a hotkey binded for a TAStudio feature or doing things on the right side of TAStudio (play pause etc).

Tests done with the lynx core:
BizHawk version, maximum FPS while minimally moving mouse, maximum FPS while manically pressing buttons
2.1.1, 211,158
2.2, 193, 150
2.2.1 ok
2.2.2, 160, 22

**Comment by vadosnaprimer** [comment]
Please make a video with 2 releases, doing similar things in them, showing the resulting slowdown in one of them. Maybe I'll be able to track it down by a simple bisect.

**Comment by meshuggahtas** [comment]
https://www.youtube.com/watch?v=QxWSDC9QTkU

**Comment by meshuggahtas** [comment]
default config + no throttling + lynx electrocop + follow cursor a new empty project in TAStudio

BizHawk 2.2.2: 85~100 fps when doing nothing. 30~50 fps when hovering the mouse over piano roll with some speed.
BizHawk 2.2.1: 90~110 fps when doing nothing. 66~88 fps when hovering the mouse over piano roll with some speed.

**Comment by meshuggahtas** [comment]
Tested the artifact, got a little FPS back, better than nothing. No need to reopen, I will buy a new PC or laptop. I would say I don't have THAT potato configuration, but it's still old. Full configuration and settings of my PC: http://speccy.piriform.com/results/VSaYiF3pztwFD7noHkcWDDS

General performance overhead still remains. The lowest number I saw was 34 (very fast mouse movement over piano roll and even clicking sometimes), but I also had a little bit higher fps like 60.
